<launch>
    <arg name="controller" default="NL" />

    <node pkg="mushr_control" type="main.py" name="controller" output="screen">
        <param name="type" value="$(arg controller)" />

        <param name="use_sim_pose" value="1" />
    </node>
</launch>
